dashboard/src/components/Config/AppRepoList/AppRepoList.tsx

```diff
@@ -117,17 +130,27 @@ class AppRepoList extends React.Component<IAppRepoListProps> {
                       repo={repo}
                       renderNamespace={renderNamespace}
                       namespace={namespace}
+                      errors={errors}
+                      validating={validating}
+                      validate={validate}
+                      secret={repoSecrets.find(secret =>
```

Comment:
So it's just the first secret which has the app repo as owner (which should be safe for now, since we won't want ownership for the registry secrets).